frontend/src/flashcards/FlashcardFront.js

```diff
@@ -28,12 +28,20 @@ const imgStyle = {
 };
 
 function FlashcardFront(props) {
+	if (props.text === "+New Card") {
```

Comment:
This is very suspicious. Components are like functions -- given props, they return some output -- and therefore should do one thing only. Whenever you have an if statement inside a function, treat it with suspicion because sometimes an if statement means a function is doing too many things.

In this case, you should make the New Card a different component and render that component instead of checking within this component. (Also, what happens if a user makes a flash card with the text "+New Card"?

Reply:
done.